digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/ChangeApi.scala

```diff
@@ -164,6 +163,44 @@ class ChangeApi(val swagger: Swagger) extends ScalatraServlet with JacksonJsonSu
     }
   }
 
+  private def isSuggested(asset: ChangedLinearAsset): Boolean = {
+    asset.linearAsset.value match {
+      case Some(Prohibitions(_, isSuggested)) => isSuggested
+      case Some(SpeedLimitValue(_, isSuggested)) => isSuggested
+      case Some(DynamicValue(x)) =>
+        x.properties.find(_.publicId == "suggest_box").flatMap(_.values.headOption) match {
+          case Some(value) => value.toString.toBoolean
+          case _ => false
+        }
+      case _ => false
+    }
+  }
+
+  private def isSuggested(asset: ChangedPointAsset): Boolean = {
+    asset.pointAsset.propertyData.find(_.publicId == "suggest_box").flatMap(_.values.headOption) match {
+      case Some(value) => value.toString.toBoolean
+      case _ => false
+    }
+  }
```

Comment:
This method is being used by pointAssetToGeoJson only. And since the way to reach the propertyData is different from linear to point maybe it's not possible to delete this method.